_people/james_pheasey.md

```diff
@@ -4,9 +4,11 @@ gender: male
 graduated: 2019
 submitted: 2019-03-13
 course:
-  - Philosophy BA
-  - Philosophy MA
+  - BA Philosophy
+  - MA Philosophy
 contact_allowed: true
+
+# SU President (2019/2020)
 ---
 
-SU President (2019/2020)
```

Comment:
Why comment this?

Reply:
if it was part of a larger Bio then i'd leave it but it seems a bit odd just being there by itself, have no strong feelings so am happy to revert

Reply:
Can it go in careers?